src/invidious/views/watch.ecr

```diff
@@ -126,7 +128,7 @@
                 <script src="/js/playlist_widget.js?v=<%= Time.utc.to_unix_ms %>"></script>
             <% end %>
 
-            <% if CONFIG.dmca_content.includes?(video.id) || CONFIG.disabled?("downloads") %>
+            <% if CONFIG.child_friendly || CONFIG.dmca_content.includes?(video.id) || CONFIG.disabled?("downloads") %>
```

Comment:
why is a child not allowed to download any video? what if he/she wants to download an educational video in order to share it with his/her friends?